src/scripts/actions/VideoActions.js

```diff
@@ -17,3 +17,8 @@ export const fetchVideo = (id: string) => async (dispatch: Dispatch<*>) => {
   console.log('Received video info')
   dispatch(videoDataLoaded(videoInfo))
 }
+
+export const setCurrentVideo = (id: string) => (dispatch: Dispatch<*>) => {
```

Comment:
There is no need to have `setCurrentVideo` right now, all it does is call `dispatch(videoDataLoaded())`. You can instead pass `videoDataLoaded` to the container